ui/imports/shared/controls/AmountInput.qml

```diff
@@ -10,14 +10,15 @@ import utils 1.0
 Input {
     id: root
 
-    property int maximumLength: 10
+    property int maximumLength: 10 + tokenDecimals
```

Comment:
I'm not sure about that.  I think the maximum allowed digits should be 10 including the decimals. Probably a question for @benjthayer ?

Reply:
Hi @noeliaSD 

I added numbers of decimals to HoldingsDropdownPage.qml and AirdropsSettingsPanelPage.qml for testing purposes

<img width="799" alt="image" src="https://github.com/status-im/status-desktop/assets/1083341/31052570-cf37-44a4-8866-5caa29a02ac8">

<img width="803" alt="image" src="https://github.com/status-im/status-desktop/assets/1083341/17b8ae68-90af-48d0-8e15-bf4963cfb545">

(I skipped PermissionsSettingsPanelPage.qml because if I follow the same approach, the UI will change a bit, so I need to investigate this further. If you don't mind, I'd suggest doing this separately)

Reply:
Thanks for adding it to `storybook`!

Reply:
> I'm not sure about that. I think the maximum allowed digits should be 10 including the decimals. Probably a question for @benjthayer ?

Regarding this comment, I've been talking with `Ben` about this digits restriction and after checking the requirements his last design/requirements are the following:

1) For community assets we can validate if the user enters a larger number of `decimal points` than the specified number of `decimal points` for that asset (specified during the community asset minting flow)
2) No other restriction on digits or decimals other than what fits in the input. 

There are 2 specific states added in the [designs](https://www.figma.com/file/qHfFm7C9LwtXpfdbxssCK3/Kuba%E2%8E%9CDesktop---Communities?type=design&node-id=52547-86075&mode=design&t=pdq48KFhbmfDZiQz-4) to cover them:

![image](https://github.com/status-im/status-desktop/assets/97019400/3910bd28-3870-4da7-9429-5e37b0b85baa)

**Summary:**
- We need to update the error message text when validating decimals.
- We don't need to add validation of 10 digits but blocking the user to input more digits than the ones that fit in the input (AND no error displayed when that happens).

Hope that's clear! :blush: 

cc: @benjthayer

Reply:
Okay, thanks for the clarification! I'll fix it

Reply:
I've updated the PR 

The udpated copy:
![Screenshot 2024-03-13 at 18 10 19](https://github.com/status-im/status-desktop/assets/1083341/bd919830-d388-4783-b611-5021155821bb)

as per discussed with @benjthayer possible solution for the Input field without the character limit:
https://github.com/status-im/status-desktop/assets/1083341/bcbf062b-6a22-45df-9321-06911a0af927
^
(the number of characters may depend on the DPI of the screen, or on custom fonts. For example, at 100% will fit 15 digits, but at 250% only 7. I would suggest not limiting the number of digits in width, because it may work differently on different computers)

Reply:
> Hi @noeliaSD please check the updated PR
> 
> > We need to update the error message text when validating decimals
> 
> The updated copy (additionally I've aligned the validation error to the left):
> 

Thanks for your update! I can see this warning on `storybook` that would worth it if you can take a look:

https://github.com/status-im/status-desktop/assets/97019400/65919c78-4559-441d-83a4-ee6a27389482

> (the number of characters may depend on the DPI of the screen, or on custom fonts. For example, at 100% will fit 15 digits, but at 250% only 7. I would suggest not limiting the number of digits in width, because it may work differently on different computers)

I've asked again to `Ben` to see how he sees the solution proposed! Thanks you!

Reply:
>  Screen.Recording.2024-03-19.at.12.20.51.mov 
> > (the number of characters may depend on the DPI of the screen, or on custom fonts. For example, at 100% will fit 15 digits, but at 250% only 7. I would suggest not limiting the number of digits in width, because it may work differently on different computers)
> 
> I've asked again to `Ben` to see how he sees the solution proposed! Thanks you!

Ok, so, _Ben_ is ok with this solution right now, but we will create a separate task to add lateral scrolling into status input so it's easier for the user to check the amount not only using the arrows: https://github.com/status-im/status-desktop/issues/14032

Reply:
the warning is fixed